schemas/ws/ws_genome_has_feature.yaml

```diff
@@ -4,8 +4,13 @@ schema:
   "$schema": http://json-schema.org/draft-07/schema#
   type: object
   description: A workspace genome has a feature.
-  required: [_from, _to]
+  required: [_from, _to, _key]
   properties:
+    _key:
```

Comment:
The RE API automatically generates the _key field for edges. It creates a hash from the _from and _to keys. Is this going to be problematic for you?

Reply:
Actually, yeah, since I need to update the `_key` definition to include the data source. That allows other data sources to add the same edge (with potentially different fields)

Reply:
That's for the f->GO edges. For the edge this comment references, it probably doesn't matter.

Reply:
Nevermind, the RE API only auto-generates a _key for edges if the user does not supply it. So I think we're fine.